twingate/internal/provider/resource/service-key.go

```diff
@@ -59,6 +65,15 @@ func (r *serviceKey) Schema(_ context.Context, _ resource.SchemaRequest, resp *r
 				Computed:    true,
 				Description: "The name of the Service Key",
 			},
+			attr.ExpirationTime: schema.Int64Attribute{
+				Optional:    true,
+				Computed:    true,
+				Description: "Specifies how many days until a Service Account Key expires. This should be an integer between 0 and 365 representing the number of days until the Service Account Key will expire. Defaults to 0, meaning the key will never expire.",
+				Default:     int64default.StaticInt64(0),
+				PlanModifiers: []planmodifier.Int64{
+					int64planmodifier.RequiresReplace(),
+				},
+			},
 			// computed
 			attr.ID: schema.StringAttribute{
```

Comment:
Lets add the raw status to computed state

Reply:
added `status` attribute

Reply:
removed `status`